client/src/store/preloader/preloaderAction.js

```diff
@@ -0,0 +1,11 @@
+import { SET_LOADING, CANCEL_LOADING } from '../actionTypes';
+
+export const setLoading = () => ({
```

Comment:
Do we really need 2 action-types since they are handled equally?

Reply:
At this stage yes, as we don't now how it will works in the future. May be we'll change this logic after we see the whole picture of how this component will be called.

Reply:
Agreed, but as a rule of thumb always develop the most straightforward solution first and then extend it if needed, not other way around.

Reply:
Ok, got it. Thanks!